# 统计学基础 01：均值与中位数

## 问题

下面是一组设备故障持续时间，单位为分钟：

```text
10、11、12、13、14
```

现在新增一次持续时间为 `100` 分钟的长时故障。

请分别计算：

- 新增长时故障前的均值和中位数
- 新增长时故障后的均值和中位数

最后根据计算结果，用一句话回答：

> 极端值对均值和中位数的影响有什么不同？

In [1]:
import pandas as pd

duration_before = pd.Series([10, 11, 12, 13, 14])

duration_after = pd.Series([10, 11, 12, 13, 14, 100])

In [6]:
# 新增长时故障前的均值和中位数

before_mean = duration_before.mean()
before_median = duration_before.median()

print(f"新增前的平均值：{before_mean}")
print(f"新增前的中位数：{before_median}")


# 新增长时故障后的均值和中位数

after_mean = duration_after.mean()
after_median = duration_after.median()

print(f"新增后的平均值：{after_mean:.2f}")
print(f"新增后的中位数：{after_median}")

新增前的平均值：12.0
新增前的中位数：12.0
新增后的平均值：26.67
新增后的中位数：12.5


## 分析结论

加入持续时间为 100 分钟的长时故障后：

- 均值从 `12.00` 上升到 `26.67`；
- 中位数只从 `12.00` 上升到 `12.50`。

这说明均值容易受到极端值影响，而中位数对极端值更加稳定。

因此，当故障时长数据中存在少量特别长的故障时，中位数通常比均值更能代表大多数故障的典型持续时间。